Classes/Settings/SettingsViewController.swift

```diff
@@ -203,7 +203,7 @@ GitHubSessionListener {
 
     private func onSignOut() {
         let title = NSLocalizedString("Are you sure?", comment: "")
-        let message = NSLocalizedString("All of your accounts will be signed out, and their bookmarks will be removed. Do you want to continue?", comment: "")
+        let message = NSLocalizedString("All of your accounts will be signed out. Do you want to continue?", comment: "")
```

Comment:
Suggestion: _You will be signed out from all of your accounts. Do you want to sign out?_ 

The two buttons are `Cancel` and `Sign out` and  imo, the message  _Do you want to sign out?_ reflects the `Sign out` action.

Reply:
@jdisho thanks for the help! I've changed it